packages/azure-themes/src/stories/Themes/Themes.tsx

```diff
@@ -39,11 +39,18 @@ import { DetailsListCompactExample } from '../components/detailsList';
 import { DatePickerBoundedExample } from '../components/dateBoundary';
 import { PivotBasicExample } from '../components/Pivots';
 import { TeachingBubbleBasicExample } from '../components/TeachingBubble';
+import { MessageBarBasicExample } from '../components/messageBar';
 
 const Example = () => (
   <Stack gap={8} horizontalAlign="center" style={{ maxWidth: 1000 }}>
     <Stack gap={8} horizontalAlign="center">
+      <DefaultButton primary={true} text="bug" />
+      <DefaultButton primary={true} disabled={true} text="bug" />
```

Comment:
Perhaps we could move those two next to buttons examples so that they're all grouped together

https://github.com/microsoft/fluentui/blob/b21429525d22b180704d71a32ceae06267473bee/packages/azure-themes/src/stories/Themes/Themes.tsx#L55-L64

Reply:
Done